test/e2e/integration/frontend-test/group.ts

```diff
@@ -101,7 +101,7 @@ describe('Group', () => {
     cy.get(appFrontend.group.newValueLabel).should('contain.text', '2. Endre verdi 1338 til');
   });
 
-  it('Validation on group', () => {
+  it.only('Validation on group', () => {
```

Comment:
```suggestion
  it('Validation on group', () => {
```

Hmm, didn't the husky hook that should prevent committing this work? :thinking: Or maybe you forced the commit without running commit hooks?

Reply:
woops 🙈 , I did not force any commit with `--force` if i remember correctly 🤔 But do think i remember being stopped when attempting a commit via VSCode and simply retrying via VSCode made it go through so that might have been it?